src/Medusa.ts

```diff
@@ -28,69 +28,67 @@
  * Load medusa and apply all components
  */
 export class Medusa {
-	readonly #express: Express;
-	readonly #rootDir: string;
+    readonly #express: Express;
+    readonly #rootDir: string;
 
-	/**
-	 * @param rootDir Directory where the `medusa-config` is located
-	 * @param express Express instance
-	 */
-	constructor(rootDir: string, express: Express) {
-		this.#express = express;
-		this.#rootDir = rootDir;
-	}
+    /**
+     * @param rootDir Directory where the `medusa-config` is located
+     * @param express Express instance
+     */
+    constructor(rootDir: string, express: Express) {
+        this.#express = express;
+        this.#rootDir = rootDir;
+    }
```

Comment:
Codacy has a fix for the issue: [Replace `····` with `↹`](https://app.codacy.com/gh/adrien2p/medusa-extender/pullRequest?prid=9061797)
```suggestion
	}
```